src/tracker.jl

```diff
@@ -518,10 +518,7 @@ end
 function Distributions.logpdf(d::Wishart, X::AbstractArray{<:TrackedMatrix})
     return logpdf(TuringWishart(d), X)
 end
-Distributions.Wishart(df::TrackedReal, S::Matrix{<:Real}) = TuringWishart(df, S)
-Distributions.Wishart(df::TrackedReal, S::AbstractMatrix{<:Real}) = TuringWishart(df, S)
 Distributions.Wishart(df::Real, S::TrackedMatrix) = TuringWishart(df, S)
-Distributions.Wishart(df::TrackedReal, S::TrackedMatrix) = TuringWishart(df, S)
```

Comment:
Same here, why can we drop it?

Reply:
It seems instead of dropping these constructors we should fix the ambiguity by adding explicit constructors for `AbstractPDMat`, similar to the ReverseDiff PR, such as
```julia
Distributions.Wishart(df::TrackedReal, S::AbstractPDMat{<:TrackedReal}) = TuringWishart(df, S) 
```

Reply:
`TrackedReal` is a subtype of `Real`, so that constructor should not be necessary?

Reply:
The problem is the ambiguity error that arises from having both `Wishart(::Real, ::AbstractMatrix{<:Real})` defined above and `Wishart(::T, ::AbstractPDMat{<:T}) where {T<:Real}` in https://github.com/JuliaStats/Distributions.jl/blob/master/src/matrix/wishart.jl#L45 since `AbstractPDMat <: AbstractMatrix`.